src/modules.js

```diff
@@ -110,9 +110,10 @@ var LibraryManager = {
         }
         libraries.push('library_noderawfs.js');
       }
-    } else {
-      libraries.push('library_wasmfs.js');
     }
+    //  else {
+    //   libraries.push('library_wasmfs.js');
+    // } TODO: populate with library_wasmfs.js later
```

Comment:
I would remove the commented out code and leave just the TODO comment just to keep the code looking cleaner.